TweetBook/Services/IIdentityService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using TweetBook.Domain;
+
+namespace TweetBook.Services
+{
+    public interface IIdentityService
+    {
+      public  Task<AuthenticationResult> RegisterAsync(string email, string password);
```

Comment:
Too many empty spaces between the access modifier and the return type.
You could enable "view whitespace" in visual studio like in here: 
![image](https://user-images.githubusercontent.com/61745471/155970186-9c076129-0c09-4c9a-a126-0de8fb3b2a19.png)